### PR TITLE
Set up CMS for Registry

### PR DESCRIPTION
**Pull request by blushi**
ref: regen-network/regen-registry#608

### CODE REVIEW

**Review by haveanicedavid — APPROVED** [review]
Looks good! I'm excited to dig into this

File: deskStructure.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import * as Structure from 'sanity-plugin-intl-input/lib/structure';
+import S from '@sanity/desk-tool/structure-builder';
 
 // default implementation by re-exporting
-export const getDefaultDocumentNode = Structure.getDefaultDocumentNode;
-export default Structure.default;
+// export const getDefaultDocumentNode = Structure.getDefaultDocumentNode;
```
Comment (mhagel) [inline deskStructure.js:5]: Keeping these commented lines intentionally?

File: deskStructure.js [hunk 2]
```diff
@@ -12,10 +13,109 @@ export default Structure.default;
...
-//   return S.list()
-//       .id('__root__')
-//       .title('Content')
-//       .items(items);
-// };
\ No newline at end of file
+export default () => {
+  // const items = Structure.getFilteredDocumentTypeListItems();
+  return S.list()
+      // .id('__root__')
```
Comment (mhagel) [inline deskStructure.js]: Keeping these commented lines intentionally?

File: schemas/objects/connectSectionLink.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+      title: 'Description',
+      name: 'description',
+      type: 'customPortableText',
+      validation: Rule => Rule.required(),
+    },
+    {
+      title: 'Icon',
+      name: 'icon',
+      type: 'image',
+      validation: Rule => Rule.required()
```
Comment (mhagel) [inline schemas/objects/connectSectionLink.js:22]: I like how you can write custom validation

**Review by mhagel — APPROVED** [review]
This is very readable and looks easy to manage. Very cool!

File: deskStructure.js (hunk 1 above)
Comment (blushi) [reply deskStructure.js:5]: yes keeping for later when we add i18n support (this is config related to the sanity i18n plugin)

File: deskStructure.js (hunk 2 above)
Comment (blushi) [reply deskStructure.js]: removed

**Pull request merged**